### PR TITLE
Fix tinting issues

### PR DESCRIPTION
**Pull request by MillerApps**
This fixes some `UIActivtityViewController` tint issues. **Please improve upon as you feel fit!**

### CODE REVIEW

File: Trust/Browser/ViewControllers/BrowserViewController.swift [hunk 1]
```diff
@@ -247,17 +247,19 @@ class BrowserViewController: UIViewController {
...
-    }
-
     private func share() {
         guard let url = webView.url else { return }
         guard let navigationController = navigationController else { return }
-        let controller = makeShareController(url: url)
-        controller.popoverPresentationController?.sourceView = navigationController.view
-        controller.popoverPresentationController?.sourceRect = navigationController.view.centerRect
-        navigationController.present(controller, animated: true, completion: nil)
+        let activityViewController = ActivityViewController(
```
Comment (vikmeup) [inline Trust/Browser/ViewControllers/BrowserViewController.swift]: can we add navigationController into initializer? so we don't forget to call this method

`makeShareController` can you make other places to use this initializer?

Comment (MillerApps) [reply Trust/Browser/ViewControllers/BrowserViewController.swift]: You want to use`makeShareController` for each `ActivityViewController`?

Comment (vikmeup) [reply Trust/Browser/ViewControllers/BrowserViewController.swift]: Yeah, that makes it more reusable

Comment (MillerApps) [reply Trust/Browser/ViewControllers/BrowserViewController.swift]: Done

**Pull request merged**